lib/widgets/trufi_drawer.dart

```diff
@@ -103,11 +103,12 @@ class TrufiDrawerState extends State<TrufiDrawer> {
             localization.menuYourPlaces,
             SavedPlacesPage.route,
           ),
+          if (cfg.configurationDrawer.definitionFeedBack != null) 
           _buildListItem(
             Icons.feedback,
             localization.menuFeedback,
             FeedbackPage.route,
-          ),
+          ) else Container(),
```

Comment:
Is this container only to show nothing? If so you could just remove the else part:
```dart
if (cfg.configurationDrawer.definitionFeedBack != null)
            _buildListItem(
              Icons.feedback,
              localization.menuFeedback,
              FeedbackPage.route,
            ),
```